src/sources/aws_kinesis_firehose/mod.rs

```diff
@@ -37,9 +37,18 @@ pub struct AwsKinesisFirehoseConfig {
     ///
     /// AWS Kinesis Firehose can be configured to pass along a user-configurable access key with each request. If
     /// configured, `access_key` should be set to the same value. Otherwise, all requests will be allowed.
+    ///
+    /// This option has been deprecated, the `access_keys` option should be used instead.
     #[configurable(metadata(docs::examples = "A94A8FE5CCB19BA61C4C08"))]
```

Comment:
```suggestion
    /// This option has been deprecated, the `access_keys` option should be used instead.
    #[configurable(deprecated)]
    #[configurable(metadata(docs::examples = "A94A8FE5CCB19BA61C4C08"))]
```

Sorry, still nitting 😄